<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResultDialog</class>
 <widget class="QDialog" name="ResultDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="labelResult10kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>180</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 10 km/h: 5,6 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult40kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>120</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 40 km/h: 9,8 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult20kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 20 km/h: 6,9 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult30kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>140</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 30 km/h: 8,6 s</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSend">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>260</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Send results</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult80kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 80 km/h: 9,8 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult70kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>60</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 70 km/h: 8,6 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult60kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 60 km/h: 6,9 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult50kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 50 km/h: 5,6 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult90kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 90 km/h: 8,6 s</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelResult100kmh">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>0</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0 - 100 km/h: 9,8 s</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
